test/integration/api_v1_test.rb

```diff
@@ -586,6 +586,27 @@ class ApiV1Test < ActionDispatch::IntegrationTest
     assert_response :unauthorized
   end
 
+  test 'add a key to a project' do
```

Comment:
I tried to write a test for unprocessable using the wrong params but the test gave an error instead.